.travis.yml

```diff
@@ -0,0 +1,33 @@
+language: c
+sudo: required
+env:
+  global:
+    - ZFS_TEST_TRAVIS_LOG_MAX_LENGHT=400
+  matrix:
+    - ZFS_TEST_TRAVIS_BUILDER_FROM='[tests/functional/acl/posix]' ZFS_TEST_TRAVIS_BUILDER_TO='[tests/functional/cli_root/zpool_import]'
+    - ZFS_TEST_TRAVIS_BUILDER_FROM='[tests/functional/cli_root/zpool_import]' ZFS_TEST_TRAVIS_BUILDER_TO='[tests/functional/cli_root/zpool_set]'
+    - ZFS_TEST_TRAVIS_BUILDER_FROM='[tests/functional/cli_root/zpool_set]' ZFS_TEST_TRAVIS_BUILDER_TO='[tests/functional/replacement]'
+    - ZFS_TEST_TRAVIS_BUILDER_FROM='[tests/functional/replacement]'
+before_install:
+- sudo apt-get -qq update
+- sudo apt-get install --yes -qq build-essential autoconf libtool gawk alien fakeroot linux-headers-$(uname -r)
+- sudo apt-get install --yes -qq zlib1g-dev uuid-dev libattr1-dev libblkid-dev libselinux-dev libudev-dev libdevmapper-dev parted lsscsi ksh attr acl nfs-kernel-server
+install:
+- git clone --depth=50 https://github.com/zfsonlinux/spl
+- cd spl
+- git checkout master
+- sh autogen.sh
+- ./configure
+- make -s -j$(nproc)
+- make pkg &> /dev/null
+- sudo dpkg -i *.deb
+- cd ..
+- sh autogen.sh
+- ./configure
+- make -s -j$(nproc)
+- make pkg &> /dev/null
+- sudo dpkg -i *.deb
+script:
+- travis_wait 50 ./scripts/zfs-tests-travis.sh
```

Comment:
Rather than add a new script for this purpose I'd prefer extending the existing `zfs-tests.sh`.  @dinatale2 already has #5775 open to allow running a single test, how about extending that PR to support running a range of tests.